src/OpenApi/Config/Spec.php

```diff
@@ -28,12 +30,21 @@
     description: 'API for Studio generated by OpenApi Generator via zircote/swagger-php',
     title: 'Studio Backend API'
 )]
+
+#[OpenApi(
+    openapi: '3.1.0'
+)]
+
 #[License(
     name: 'GNU General Public License version 3 & Pimcore Commercial License',
-    identifier: 'GPLv3 & PCL',
     url: 'https://www.pimcore.org/license'
 )]
 
+#[Server(
+    url: 'http://localhost:6001',
```

Comment:
We should not set this as a fixed value.
Like we discussed, we can inject it in the OpenApi Service. 
I would remove it for now and then add another PR where you can configure this.